tests/kernel/wave/attention/extend_attention_test.py

```diff
@@ -464,7 +465,7 @@ def testExtendRpeAttention(
         rpe_bias,
         max_rpe_context_length,
     ) = create_inputs(shape, dtype)
-    shape.max_seq_len = max_len_extend
```

Comment:
What does this change do?

Reply:
I wanted to make attentionshape hashable and so now that it is hashable, you can only update the value by making a copy and dataclass has a convenient function replace that lets you copy a dataclass and only replace specified fields.